Dockerfile

```diff
@@ -21,3 +21,4 @@ RUN conda update -n usi -c anaconda openssl
 
 COPY . /app
 WORKDIR /app
+RUN pip install -e .
```

Comment:
might be able to skip all of those other conda install commands now that this is here. You could probably remove the `-e` editable flag too